Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -3791,6 +3792,83 @@ def com_google_fonts_check_name_family_and_style_max_length(ttFont):
     yield PASS, "All name entries are good."
 
 
+@check(
+    id='com.google.fonts/check/has_unacceptable_control_chars',
+    conditions=['are_ttf']
+)
+def com_google_fonts_check_has_unacceptable_control_chars(ttFonts):
+  """Does font file include unacceptable control character glyphs?
+
+  Args:
+      ttFonts (list): list of one or more fontTools.ttLib.TTFont objects
+  """
+  # list of unacceptable control character glyph names
+  # definition includes the entire control character Unicode block except:
+  #    - .null (U+0000)
+  #    - CR (U+000D)
```

Comment:
This would be good to be placed on the rationale field of the check. Would also be good to mention why these exceptions are accepted. (also we actually require these on another check, by the way)

Another piece of info for the rationale field would be a short description of the Google Fonts problem that inspired us to add this check. It may help us in the future in case we may want to double-check the reasoning behind this to either deprecate the check (if it ends up being moot) or to improve the description once we figure out more details about it.

Reply:
I don't have any basis for the exceptions on these control chars.  I'm not aware of any spec for this issue.  This decision was based on a review of other font files and Marc's input.

Re: the problem that led to this check.  I don't have this information.  The bug report went directly to Dave C and he requested that the new FB test.

Adding rationale field to the test based on your request here.

Reply:
done!